Library/brew.rb

```diff
@@ -50,6 +50,20 @@
   EOABORT
 end
 
+if OS.mac? && MacOS.version == "10.11" && MacOS::Xcode.version < "7.0"
```

Comment:
I thought @mikemcquaid wanted it runtime for `brew`, rather than just on `install`, but I could have misunderstood :smile_cat:.

Agreed that the conditional is too simplistic in this situation; Should've realised that before I pushed really. Will tune it up to cover the wider scope, thanks :)